docs/platform/deployservices.md

```diff
@@ -1,13 +1,311 @@
+please refer to [this section](https://github.com/smartsdk/smartsdk-recipes/tree/master/recipes/utils/mongo-replicaset)
+ of the SmartSDK recipes documentation.
+
+### Using Portainer
+
+Coming Soon!
+
+## Deploy a Highly Available Context Broker
+
+### From the Command Line
+
+Orion needs a MongoDB database to work. If you have been following this guide,
+you should have already one replicaset running. The default configurations in
+the settings files assume we will use those mongo instances.
+
+To deploy Orion, please execute the following commands...
+
+```
+$ cd data-management/context-broker/ha/
+$ source settings.env  # In Windows, simply execute settings.bat instead.
+$ docker stack deploy -c docker-compose.yml orion
+$ cd ../../../
+```
+
+The first time this will take a while until the images are pulled to the nodes.
+With command `docker service ls` you can check if all the replicas are up or not.
+After some minutes, when all replicas are up, you can check that Orion was
+successfully deployed by checking its logs...
+
+```
+$ docker service logs orion_orion
+orion_orion.2.yu0h9clplwms@ms-worker0    | time=Thursday 26 Apr 08:05:32 2018.325Z | lvl=INFO | corr=N/A | trans=N/A | from=N/A | srv=N/A | subsrv=N/A | comp=Orion | op=contextBroker.cpp[1835]:main | msg=Orion Context Broker is running
+orion_orion.2.yu0h9clplwms@ms-worker0    | time=Thursday 26 Apr 08:05:32 2018.331Z | lvl=INFO | corr=N/A | trans=N/A | from=N/A | srv=N/A | subsrv=N/A | comp=Orion | op=mongoConnectionPool.cpp[217]:mongoConnect | msg=Successful connection to database
+```
+
+If you can see messages like those (Orion Running and connected to database),
+your **Orion** is ready! However, if you still have issues or want to understand
+ how this works in more detail, please refer to [this section](https://smartsdk.github.io/smartsdk-recipes/data-management/context-broker/ha/readme/)
+ of the SmartSDK recipes documentation.
+
+### Using Portainer
+
+Coming Soon!
+
+## Deploy a Highly Available QuantumLeap
+
+Quantumleap is built on top of a CrateDB cluster. Some values of the
+configuration will depend on the number of nodes you have in your swarm cluster.
+
+The default values will deploy a 1-node cluster, to make sure it works as default
+for new users. However, if you know you cluster has more nodes, you can adjust
+those values following the documentation. For example, with a 3-nodes cluster,
+you can update these values from the default `settings.env`:
+
+```
+export EXPECTED_NODES=3
+export RECOVER_AFTER_NODES=1
+export MINIMUM_MASTER_NODES=2
+```
+
+### Prerequisites
+
+Crate needs a minimum of 262144 for the mmap count, which by default is lower in
+Swarm nodes. Thus, before attempting to launch this service, you should run the
+following command in each of your swarm nodes.
+
+Assuming you followed the miniswarm example, you can do...
+
+```
+$ docker-machine ssh ms-manager0
+docker@ms-manager0:~$ sudo sysctl -w vm.max_map_count=262144
```

Comment:
@dancn is this something we should take care in the platform docs maybe?

Reply:
@taliaga I will take care of adding this in the platform docs and to add this line in our custom deployer script.

Reply:
ping me when you have news so I recall to remove it from here afterwards :)